.travis.yml

```diff
@@ -0,0 +1,9 @@
+language: node_js
+node_js:
+  - "11"
+cache:
+  directories:
+  - node_modules
+script:
+  - npm test
+  - npm ci
```

Comment:
Good. This data shows that only two files were changed. Don't you think it's important to include a badge which shows if our build is passing or failing in the README.md file?

Reply:
Oh thanks, would add that now